### PR TITLE
Add pacs002 e2e tests

### PR DESCRIPTION
**Pull request by theolextego**

### CODE REVIEW

**Comment by uche-lextego** [comment]
@theolextego , the typology host should use the gateway URL. I probably need to update mine too. 😄

**Review by uche-lextego — CHANGES_REQUESTED** [review]
@theolextego , the typology host should use the gateway URL. I probably need to update mine too. 😄

File: CRSP-Tests.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+							"                \"txTp\": \"pain.001.001.11\",",
+							"                \"channels\": [",
+							"                    {",
+							"                        \"id\": \"001@1.0\",",
+							"                        \"host\": \"http://openfaas:8080\",",
+							"                        \"cfg\": \"1.0\",",
+							"                        \"typologies\": [",
+							"                            {",
+							"                                \"id\": \"028@1.0\",",
+							"                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",",
```
Comment (uche-lextego) [inline CRSP-Tests.postman_collection.json:65]: I've seen the comment functionality @theolextego

**Review by uche-lextego — APPROVED** [review]
LGTM

**Review by johanfol — APPROVED** [review]

**Pull request merged**